### DISCUSSION TITLE
CogVLM bilingual version whether will be open source?

### DISCUSSION THREAD

**Discussion by especiallyW** [THUDM/CogVLM#249] [Q&A] [answered]
据我所知的是，目前开源的CogVLM/CogAgent，其语言模型都是基于Hugging face上的vicuna-7b-v1.5，而该模型没有太多得涉及中文数据集。（若我的理解有误，请指正。）

所以，CogVLM双语版是否会开源？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
暂时没有开源，我们有提供API和商业版本
